include/RAJA/exec-openmp/scan_openmp.hxx

```diff
@@ -82,6 +82,7 @@ void inclusive_scan_inplace(const omp_parallel_for_exec&,
                             ValueT v)
 {
   using Value = typename std::decay<decltype(*std::declval<Iter&>())>::type;
+  // using Value = typename std::iterator_traits <Iter>::value_type;
```

Comment:
Here's a gist that works. It must be something with my testing framework.

https://gist.github.com/willkill07/b8414947e655e8a014040b6db3f81959